tests/dialogBoxes.spec.ts

```diff
@@ -28,6 +29,6 @@ test.describe("Dialog Boxes", async () => {
       .last()
       .getByRole("button", { name: "Delete" })
       .click();
-    await expect(page.locator("table tr").last()).not.toHaveText("pig");
+    await expect(petTypesTableInputField.last()).not.toHaveValue("pig");
```

Comment:
Naming :)
This locator returns not a single input field, it returns all input fields.

Also the `last()` method can be part of the same locator, so you don't need to call this method two times in the code.

Reply:
I changed the constant name to plural.

Unfortunately, for some reason, it doesn’t work without the last() method :( It returns the entire list.

Reply:
Did you update the locator like this?
`page.locator("table tr td input").last()`

Reply:
Sorry I missed it! Now I understand what did you mean!! Thanks a lot! Fixed!